docs/getting-started/v3-migration.md

```diff
@@ -103,6 +103,7 @@ Animation system was completely rewritten in Chart.js v3. Each property can now
 * `helpers.findIndex`
 * `helpers.findNextWhere`
 * `helpers.findPreviousWhere`
+* `helpers.extend` use `Object.assign` instead
```

Comment:
nit: add a period and capitalize "use" to match the text for "afterScaleUpdate". I.e.:

```
* `helpers.extend`. Use `Object.assign` instead
```